language-server/modules/langserver-core/src/test/resources/completion/expression_context/config/mapping_expr_ctx_config5.json

```diff
@@ -15,7 +16,7 @@
           "value": "**Package:** _._  \n  \n  \n  \n  \n**Return** `int`   \n  \n"
         }
       },
-      "sortText": "C",
+      "sortText": "AC",
```

Comment:
In this case, we should sort variable (string type), functions (with string return type) first. Int variables should come next.

Reply:
https://github.com/ballerina-platform/ballerina-lang/issues/39408